drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -649,6 +649,9 @@ static void ucpd_manage_tx(struct alert_info *info)
 		 */
 		if (atomic_test_and_clear_bit(&info->evt, UCPD_EVT_TX_MSG_SUCCESS)) {
 			ucpd_set_tx_state(info->dev, STATE_WAIT_CRC_ACK);
+			/* Start the GoodCRC RX Timer */
+			data->goodcrc_rx_timer_expired = false;
```

Comment:
Instead of creating a separate variable to track timer expiration, I think you can just call `k_timer_status_get()` to check if the timeout expired.

Reply:
done